File: solutions/java/darts/1/src/main/java/Darts.java
class Darts {
    int score(double xOfDart, double yOfDart) {
        
        int points = 0;
        double cA = xOfDart * xOfDart;
        double cB = yOfDart * yOfDart;
        double h  = Math.sqrt(cA + cB); 
        
        if(h > 10.0) {
            points = 0;
        } else if(h <= 10 && h > 5) {
            points = 1;
        } else if(h <= 5 && h > 1) {
            points = 5;
        } else {
            points = 10;
        }
            
        return points;
    }
}

File: solutions/c/darts/1/darts.c
#include "darts.h"

uint8_t score(coordinate_t landing_position) {
    
    uint8_t points = 0;
    float xOfDart = landing_position.x;
    float yOfDart = landing_position.y;
    
    float h = sqrt((xOfDart * xOfDart) + (yOfDart * yOfDart));
  
    if(h > 10) {
        points = 0;
    } else if(h > 5) {
        points = 1;
    } else if(h > 1) {
        points = 5;
    } else{
        points = 10;
    }

    return points;
}